website/src/get-started/wordcount-example.md

```diff
@@ -102,13 +102,151 @@ To view the full code in Go, see
 * Writing output (in this example: writing to a text file)
 * Running the Pipeline
 
+Here is how a basic implementation of WordCount looks like.
```

Comment:
same here re: mixing code in .py file and colab that doesn't match -- can we move to the bottom of the Minimal WordCount section (after Running the pipeline) in its own section? could use similar section heading/wording as in other comments. if we eventually have all of this have the exact same code,  we could possibly go back and interleave them at that point.